CopyKeys.ps1

```diff
@@ -23,7 +23,8 @@ param(
     [switch]$ForceDebug)
 
 ### Checking for module versions and assemblies.
-#Requires -Modules @{ ModuleName="AzureRM"; ModuleVersion="6.8.1" }
+Import-Module AZ
+#Requires -Modules @{ ModuleName="Az"; ModuleVersion="1.8.0" }

```

Comment:
Am unable to run this script on a fresh powershell window due to the following error

> The script 'Copy-Keys-Az.ps1' cannot be run because the following modules that are specified by
the "#requires" statements of the script are missing: Az.

This is most likely due to the #requires statement. By default, such statements run before the script execution and so will run before the Import-Module Az statement. 
As per the Microsoft docs
> Placing a #Requires statement inside a function does NOT limit its scope. All #Requires statements are always applied globally, and must be met, before the script can execute

This error goes away on explicitly import the module.